libcontainer/nsenter/nsexec.c

```diff
@@ -754,6 +759,18 @@ void nsexec(void)
 						bail("failed to set process as dumpable");
 				}
 			}
+			/*
+			 * Unshare all of the namespaces. Now, it should be noted that this
+			 * ordering might break in the future (especially with rootless
+			 * containers). But for now, it's not possible to split this into
+			 * CLONE_NEWUSER + [the rest] because of some RHEL SELinux issues.
+			 *
+			 * Note that we don't merge this with clone() because there were
+			 * some old kernel versions where clone(CLONE_PARENT | CLONE_NEWPID)
+			 * was broken, so we'll just do it the long way anyway.
+			 */
+			if (unshare(config.cloneflags & ~CLONE_NEWCGROUP) < 0)
+				bail("failed to unshare namespaces");
```

Comment:
This hunk ended up in the wrong place. It should _replace_ a similar one, but without ` & ~CLONE_NEWCGROUP`.